lib/screenplay/suppressed_predictions.ex

```diff
@@ -37,6 +40,73 @@ defmodule Screenplay.SuppressedPredictions do
     |> Repo.all()
   end
 
+  @doc """
+  Get all suppressed predictions modified for transit data
+  """
+  @spec get_all_suppressed_predictions_for_data() :: [
+          %{
+            route_id: String.t(),
+            stop_id: String.t(),
+            direction_id: integer(),
+            suppression_type: :terminal | :stop | :none
+          }
+        ]
+  def get_all_suppressed_predictions_for_data do
+    suppressed_predictions = get_all_suppressed_predictions()
+
+    Places.get()
+    |> Enum.flat_map(fn
+      %Screenplay.Places.Place{id: "place-jfk"} ->
+        Enum.map(
+          PredictionSuppressionUtils.jfk_umass_child_stop_data(),
+          fn %{
+               route_id: route_id,
+               stop_id: child_stop_id,
+               direction_id: direction_id
+             } ->
+            %{
+              stop_id: child_stop_id,
+              route_id: route_id,
+              direction_id: direction_id,
+              suppression_type:
+                PredictionSuppressionUtils.suppression_type(
+                  suppressed_predictions,
+                  child_stop_id
+                )
+            }
+          end
+        )
+
+      %Screenplay.Places.Place{id: place_stop_id, screens: screens} ->
+        Enum.flat_map(screens, fn
+          %Screenplay.Places.Place.PaEssScreen{routes: routes} ->
```

Comment:
We shouldn't be limiting this to places with PA/ESS, with the exception of the SL which looks at only the stations currently shown in Signs UI. We can probably extract the logic in the SuppressedPrediction validation code for filtering the places list down to the ones we care about, since we want to use the same list here.

Reply:
Got it - in this particular case, is it still safe to be mapping to all of the `routes` that exist under `screens` and validating based on those? (will this just effectively filter out to the PaEss screens again?) Or should we be validating on the routes that exist on the top level place and do some logic checking for Green Line branches and Silver line?

Reply:
I guess my question kind of boils down to - if there's no PA/ESS screen, is there a good way for us to get the different routes/direction ids? Or do we have to pull that data from elsewhere?

Reply:
Yeah, the we can use the list of `routes` on the place itself for everything except Silver line. For SL, we'll need to dig into the PA/ESS signs, since the `routes` field just rolls everything up as `Silver`

Reply:
Got it - and for direction id, will it _always_ have a corresponding 0/1 direction except at terminals?